### PR TITLE
[Term Entry] PHP Math Functions ceil() #2061

### PR DESCRIPTION
**Pull request by d1r0l**
### Description

Added new term entry for PHP math function ceil()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/ceil/ceil.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Syntax
+
+```pseudo
+$result = ceil(n);
+```
+
+Where `n` is the fractional number to find the next highest integer value of.
+
+## Example
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/ceil/ceil.md:26]: Generally we want a sentence describing what the example does, and output shown not in comments, but in a separate ` ```shell ` block afterwards (introduced by a sentence like, "This example results in the following output").

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@d1r0l I did a first review. Good entry! I just had one comment about expanding your example section.

**Comment by d1r0l** [comment]
@SSwiniarski Thanks for a review and a request! I fixed the Example block according to the recommendation so please review again.

File: content/php/concepts/math-functions/terms/ceil/ceil.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/ceil/ceil.md:26]: Still would like to have a sentence introducing the example and what it does.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@d1r0l Still part of my last comment that needs to be addressed.

**Review by SSwiniarski — APPROVED** [review]
@d1r0l Looks good for review number two.

**Comment by d1r0l** [comment]
@SSwiniarski Thank you! Sorry for misunderstanding first time.

File: content/php/concepts/math-functions/terms/ceil/ceil.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+This example demonstrates the usage of ceil function with three different arguments and outputs results of each in a new line:
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/ceil/ceil.md:28]:
```suggestion
This example demonstrates the usage of the `ceil()` function with three different arguments, and outputs results of each on a new line:
```

File: content/php/concepts/math-functions/terms/ceil/ceil.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Where `n` is the fractional number to find the next highest integer value of.
+
+## Example
+
+This example demonstrates the usage of the `ceil()` function with three different arguments, and outputs results of each on a new line:
+
+```php
+<?php
+  echo ceil(9.4)."\n";
+  echo ceil(6.01)."\n";
```
Comment (yangc95) [inline content/php/concepts/math-functions/terms/ceil/ceil.md:33]:
```suggestion
  echo ceil(9.4)."</br>";
  echo ceil(6.01)."</br>";
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Congrats 🥳 @d1r0l live link: https://www.codecademy.com/resources/docs/php/math-functions/ceil

Screenshot 2023-05-26 at 3 35 09 PM

It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
To add your GitHub username to your Codecademy profile:
1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
githubusername

Make sure to set your visibility to “everyone”:

Screenshot 2023-04-10 at 4 07 30 PM

2.) Then log out and log back into Codecademy using your GitHub login:
githublogin

If the steps above do not work, it may be due to an email or local configuration issue.
In addition, we may reach out in the future if any adjustments to these entries are required.


cc: @SSwiniarski

**Pull request merged**